src/sql/parse.py

```diff
@@ -65,6 +71,18 @@ def parse(cell, config):
         if len(pieces) == 2:
             return result
         cell = pieces[2]
+    # handle flexible spacing around =<<
+    elif len(pieces) > 1 and pieces[1] == "=<<":
+        result["result_var"] = pieces[0]
+        result["return_result_var"] = True
+        cell = pieces[2]
+
+    # handle flexible spacing around =
+    pieces = cell.split(None, 3)
+    if len(pieces) > 1 and pieces[1] == "=" and pieces[2] == "<<":
+        result["result_var"] = pieces[0]
+        result["return_result_var"] = True
+        cell = pieces[3]
```

Comment:
Is it possible to generalize these 3 code blocks? the logic seems to repeat itself.